Library/Formula/luabind.rb

```diff
@@ -58,6 +58,22 @@ def install
     end
     args << "--prefix=#{prefix}"
     system "bjam", *args
+
+    (lib/"pkgconfig/luabind.pc").write pc_file
+  end
+
+  def pc_file; <<-EOS.undent
```

Comment:
Yep, this pattern is used through most other recipes. I actually asked someone to change it last time I just put the string in the install method.